content/blog/2023/05/24/2023-05-24-ash-sxn-introduction-blog-post.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Introducing Ashutosh Saxena as the GSoC 2023 Contributor Working on Docker based jenkins quickstart examples 
+tags:
+- gsoc
+- gsoc2023
+- docker
+authors:
+- ash-sxn
+opengraph:
+  image: /images/gsoc/opengraph.png
+sig:
+- gsoc
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+Hello, Jenkins community! I am really excited to be a GSoC 2023 contributor for Docker-based Jenkins quickstart examples project. In this blog post, I will take you through the details of the project in the following: 
+
+== Project Description
+
+The Docker based jenkins quickstart examples project aims to simplify Jenkins integration with Docker by providing user-friendly Docker Compose files and updating the documentation.
```

Comment:
I have a doubt with the wording of the goal. "integration" means everything and nothing. IMHO, the goal of the project is the provide a quick way to evaluate and get familiar with Jenkins. It is not aimed to provide a production grade setup. The wording of the introduction is key. I recommend some polishing here.